app/assets/javascripts/admin/addon/components/admin-report-table.js

```diff
@@ -76,6 +79,12 @@ export default class AdminReportTable extends Component {
     });
   }
 
+  @discourseComputed("model.total", "model.computedLabels")
+  formattedTotal(total, labels) {
+    const totalLabel = labels[labels.length - 1];
```

Comment:
we can use `at()` now!

```suggestion
    const totalLabel = labels.at(-1);
```

Reply:
![wow-yes](https://github.com/user-attachments/assets/11969785-7e66-4a6e-9250-91721aab0e97)